app/scripts/lib/fxa-client.js

```diff
@@ -692,13 +673,17 @@ define(function (require, exports, module) {
      *   * {Boolean} ok - true if user can send an SMS
      *   * {String} country - user's country
      */
-    smsStatus: withClient((client, sessionToken, options) => {
-      return client.smsStatus(sessionToken, options);
-    }),
+    smsStatus: createClientDelegate('smsStatus'),
+
+    recoveryEmails: createClientDelegate('recoveryEmails'),
+
+    recoveryEmailCreate: createClientDelegate('recoveryEmailCreate'),
+
+    recoveryEmailDestroy: createClientDelegate('recoveryEmailDestroy'),
+
+    resendEmailCode: createClientDelegate('resendEmailCode'),
 
-    deleteEmail: withClient((client, sessionToken, email) => {
-      return client.deleteEmail(sessionToken, email);
```

Comment:
@vbudhram, @vladikoff - I don't see this method in the actual fxa-js-client anywhere. Is this supposed to be here?